CHANGELOG.md

```diff
@@ -28,6 +28,11 @@
 * Emit a deprecation warning when passing a `sassIndex` with units to
   `Value.sassIndexToListIndex()`. This will be an error in Dart Sass 2.0.0.
 
+### JS API
+
+* Importer results now validate whether `contents` is actually a string type.
+* Importer result argument errors are now rendered correctly.
```

Comment:
Nit: We usually include a blank line between bullet points to help with plain-text readability.